server/bin/pbench-unpack-tarballs.sh

```diff
@@ -136,7 +136,7 @@ function  move_symlink() {
     mv ${ARCHIVE}/${hostname}/${linksrc}/${resultname}.tar.xz ${ARCHIVE}/${hostname}/${linkdest}/${resultname}.tar.xz
     status=${?}
     if [[ ${status} -ne 0 ]]; then
-        echo "${TS}: Cannot move symlink ${ARCHIVE}/${hostname}/${resultname}.tar.xz from ${linksrc} to ${linkdest}: code ${status}" | tee -a ${mail_content} >&4
+        status_append_and_log "${TS}: Cannot move symlink ${ARCHIVE}/${hostname}/${resultname}.tar.xz from ${linksrc} to ${linkdest}: code ${status}" ${mail_content} error
```

Comment:
With the suggestion to use `log_error` as above, this would become:

```
log_error "${TS}: Cannot move symlink ${ARCHIVE}/${hostname}/${resultname}.tar.xz from ${linksrc} to ${linkdest}: code ${status}" "${mail_content}"
```